Applications/Spire/Source/Ui/OverlayPanel.cpp

```diff
@@ -45,6 +45,55 @@ namespace {
   }
 }
 
+class OverlayPanel::ParentMoveObserver : public QObject {
```

Comment:
What functionality does this provide that the ```GlobalPositionObserver``` does not?

Reply:
The observer provides the same functionality as the `GlobalPositionObserver`.  I will replace it with `GlobalPositionObserver`.